src/main/java/oshi/hardware/platform/linux/LinuxNetworks.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.platform.linux;
+
+import java.io.IOException;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import oshi.hardware.common.AbstractNetworks;
+import oshi.hardware.stores.HWNetworkStore;
+import oshi.util.FileUtil;
+
+/**
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class LinuxNetworks extends AbstractNetworks {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LinuxNetworks.class);
+
+    private void setNetworkStats(HWNetworkStore netstore) {
+        String txBytesPath, rxBytesPath, txPacketsPath, rxPacketsPath;
+        List<String> read;
+
+        txBytesPath = "/sys/class/net/" + netstore.getName() + "/statistics/tx_bytes";
```

Comment:
These strings should probably be constants.  You might try a setup like:
  `private static final String TX_BYTES_FORMAT = "/sys/class/net/%s/statistics/tx_bytes";
`And then, to use,
  `txBytesPath = String.format(TX_BYTES_FORMAT, netstore.getName());`